make docker-services fails because zauth.h is missing

### $ `make docker-services`

```
Step 6/13 : COPY --from=libzauth-builder /usr/include/zauth.h /usr/include/zauth.h
COPY failed: stat /var/lib/docker/overlay2/380620e81dd76b3ea925a64c6b8f3dedf7bf8c612402cefb1e4bb8798e2d1a02/merged/usr/include/zauth.h: no such file or directory
Makefile:124: recipe for target 'docker' failed
make[1]: *** [docker] Error 1
make[1]: Leaving directory '/home/hermes/projects/wire-server/services/nginz'
Makefile:124: recipe for target 'docker-services' failed
make: *** [docker-services] Error 2
```

I should note that I'm running `make docker-deps docker-intermediate docker-services` when getting this error and I am **not** making `docker-builder` locally. Maybe the issue is with the docker-builder image being pulled from quay.io?
### On latest master branch:
```
$ docker images | grep quay.io/wire | grep latest
quay.io/wire/zauth                 latest              a7d688cf2070        41 minutes ago      55.8MB
quay.io/wire/spar-schema           latest              f687e464908b        41 minutes ago      79.6MB
quay.io/wire/spar-integration      latest              d76af69bc514        41 minutes ago      111MB
quay.io/wire/spar                  latest              5e39471f8c34        41 minutes ago      108MB
quay.io/wire/proxy                 latest              5016f4c84d28        41 minutes ago      74.8MB
quay.io/wire/cannon                latest              80297326627c        41 minutes ago      74.9MB
quay.io/wire/galley-journaler      latest              6ed8f2930f5f        42 minutes ago      90.7MB
quay.io/wire/galley-schema         latest              245c39d72894        42 minutes ago      77.5MB
quay.io/wire/galley-integration    latest              58ab0537d0fd        42 minutes ago      89MB
quay.io/wire/galley                latest              e15608169a59        42 minutes ago      96MB
quay.io/wire/gundeck-schema        latest              3a56360f063d        42 minutes ago      79.6MB
quay.io/wire/gundeck-integration   latest              99c75142268c        42 minutes ago      85.3MB
quay.io/wire/gundeck               latest              9c150729b605        42 minutes ago      92.2MB
quay.io/wire/brig-index            latest              5495ff980a09        42 minutes ago      83.6MB
quay.io/wire/brig-schema           latest              51c740da95e7        42 minutes ago      79.6MB
quay.io/wire/brig-integration      latest              46d5880ada6f        42 minutes ago      117MB
quay.io/wire/brig                  latest              46687107f719        42 minutes ago      118MB
quay.io/wire/alpine-intermediate   latest              a1b61d0ccafe        42 minutes ago      1GB
quay.io/wire/alpine-deps           latest              9f844d0b8151        About an hour ago   47.4MB
quay.io/wire/alpine-builder        latest              f5dac9c01009        2 days ago          9.22GB
```
### On #600 :
```
$ make docker-deps docker-intermediate docker-services
...
Step 6/13 : COPY --from=libzauth-builder /usr/include/zauth.h /usr/include/zauth.h
COPY failed: stat /var/lib/docker/overlay2/40aacb5ce0470ad20794fb206aa41273fa91a4f5d8087d29a12270ac4fa72ce6/merged/usr/include/zauth.h: no such file or directory
Makefile:124: recipe for target 'docker' failed
make[1]: *** [docker] Error 1
make[1]: Leaving directory '/home/hermes/projects/wire-server/services/nginz'
Makefile:124: recipe for target 'docker-services' failed
make: *** [docker-services] Error 2
```
```
$ docker images | grep quay.io/wire
quay.io/wire/zauth                 latest              1efa8b850341        3 minutes ago       55.8MB
quay.io/wire/spar-schema           latest              d193e1622326        4 minutes ago       79.6MB
quay.io/wire/spar-integration      latest              e663649d29b5        4 minutes ago       111MB
quay.io/wire/spar                  latest              4d20365c3080        4 minutes ago       108MB
quay.io/wire/proxy                 latest              7795cd015c75        4 minutes ago       74.8MB
quay.io/wire/cannon                latest              d6f520a8640d        4 minutes ago       74.9MB
quay.io/wire/galley-journaler      latest              1a4a1ce0b6d8        4 minutes ago       90.7MB
quay.io/wire/galley-schema         latest              108062799b17        4 minutes ago       77.5MB
quay.io/wire/galley-integration    latest              019bbf7a04fe        4 minutes ago       89MB
quay.io/wire/galley                latest              2b717ab406da        4 minutes ago       96MB
quay.io/wire/gundeck-schema        latest              68aa1b461a80        4 minutes ago       79.6MB
quay.io/wire/gundeck-integration   latest              a77aa0e3d094        4 minutes ago       85.3MB
quay.io/wire/gundeck               latest              616114a062d3        4 minutes ago       92.2MB
quay.io/wire/brig-index            latest              30827d5923d9        4 minutes ago       83.6MB
quay.io/wire/brig-schema           latest              05bc36f32b58        4 minutes ago       79.6MB
quay.io/wire/brig-integration      latest              59eb89a61a6b        4 minutes ago       117MB
quay.io/wire/brig                  latest              25e8cb2a9b38        4 minutes ago       118MB
quay.io/wire/alpine-intermediate   latest              6ca3e0e44d80        4 minutes ago       1GB
quay.io/wire/alpine-deps           latest              fec2aea92c30        17 minutes ago      47.4MB
quay.io/wire/alpine-builder        latest              f5dac9c01009        2 days ago          9.22GB
```

--- a/services/nginz/Dockerfile
+++ b/services/nginz/Dockerfile
@@ -4,7 +4,6 @@ FROM alpine:3.8 as libzauth-builder
 # Compile libzauth
 COPY libs/libzauth /src/libzauth
 RUN cd /src/libzauth/libzauth-c \
-    && export PREFIX=/usr \
     && apk add --no-cache make bash cargo libsodium-dev \
     && make install
 
@@ -14,9 +13,9 @@ FROM alpine:3.8
 ENV NGINX_VERSION 1.14.2
 
 # Install libzauth
-COPY --from=libzauth-builder /usr/include/zauth.h /usr/include/zauth.h
-COPY --from=libzauth-builder /usr/lib/libzauth.so /usr/lib/libzauth.so
-COPY --from=libzauth-builder /usr/lib/pkgconfig/libzauth.pc /usr/lib/pkgconfig/libzauth.pc
+COPY --from=libzauth-builder /usr/local/include/zauth.h /usr/local/include/zauth.h
+COPY --from=libzauth-builder /usr/local/lib/libzauth.so /usr/local/lib/libzauth.so
+COPY --from=libzauth-builder /usr/local/lib/pkgconfig/libzauth.pc /usr/local/lib/pkgconfig/libzauth.pc
 
 COPY services/nginz/third_party /src/third_party
 

